FlightJournal.Web/Models/TrainingLogViewModel.cs

```diff
@@ -22,9 +24,9 @@ public class TrainingDataWrapper
         internal TrainingDataWrapper(FlightContext db, int pilotId, int instructorId, Flight flight, int trainingProgramId)
         {
             FlightId = flight.FlightId;
-            PilotFlights = db.Flights.Where(x => x.PilotId == pilotId).Select(x => new PilotFlightView{FlightId = x.FlightId, Timestamp = x.Departure ?? x.Date}).OrderBy(v=>v.Timestamp).ToList();
-            FlightAnnotations = PilotFlights.SelectMany(x => db.TrainingFlightAnnotations.Where(y => y.FlightId == x.FlightId).OrderBy(y => x.Timestamp)).ToList();
-            AppliedExercises = PilotFlights.SelectMany(x => db.AppliedExercises.Where(y => y.FlightId == x.FlightId).OrderBy(y => x.Timestamp)).ToList();
+            PilotFlights = db.Flights.Where(x => x.PilotId == pilotId).Select(x => new PilotFlightView{Flight = x, Timestamp = x.Departure ?? x.Date}).OrderBy(v=>v.Timestamp).ToList();
```

Comment:
check for deleted ? (eller overser jeg den et andet sted)

Reply:
Den var med i det foregåede PR - er nu merget ind.